stdlib/public/TensorFlow/Ops.swift

```diff
@@ -677,15 +681,30 @@ public extension Tensor {
 }
 
 internal extension Tensor where Scalar : TensorFlowFloatingPoint {
+  @inlinable
+  static func _vjpInit(
+    concatenating tensors: [Tensor<Scalar>], alongAxis axis: Int
+  ) -> (Tensor, (Tensor) -> Array<Tensor<Scalar>>.DifferentiableView) {
+    let splits = Tensor<Int32>(tensors.map { $0.shapeTensor[axis] })
+    return (.init(concatenating: tensors, alongAxis: axis), { v in
```

Comment:
For consistency with the rest of the APIs, I'd prefer using the type name unless the type is horrendously long. WDYT?

```suggestion
    return (Tensor(concatenating: tensors, alongAxis: axis), { v in
```

Reply:
I also prefer the type name!